src/components/comment-input.tsx

```diff
@@ -72,6 +73,10 @@ export const CommentInput: React.VoidFunctionComponent = () => {
     setNewCommentText(event.target.value);
   };
 
+  if (userIsLoading) {
+    return <></>;
```

Comment:
Should we write Loading or show some spinner here?

Reply:
Nice idea! I created https://github.com/dtpstat/website/issues/42 to avoid further changes to this PR.